tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -445,10 +400,82 @@ def test_is_job_in_status_table(populate_status_tracking_table):
     assert result
 
     result = is_job_in_status_table(
+        session=session,
         instrument="swapi",
         data_level="l1b",
         descriptor="sci",
         start_date="20100101",
         version="v001",
     )
     assert not result
+
+
+@pytest.mark.skipif(
+    not POSTGRES_AVAILABLE, reason="Only postgres supports partial unique indexes."
+)
+# Loop over all combinations of status attempts that should fail
+@pytest.mark.parametrize(
```

Comment:
This is a great test for the unique constraint, can you add a test for `update_status_table` that also checks for failed duplicates?

Reply:
Yes, comes in the next PR I believe?